caffe2/python/tutorials/py_gen/MNIST.py

```diff
@@ -448,27 +431,31 @@ def AddBookkeepingOperators(model):
 pyplot.title('Prediction for the first image')
 
 
-# In[15]:
+# For convolutional models we can also see how they "think", i.e. which features they come up with. Instead of fetching learned weights, which can make less sense to a human, we fetch results of colvolving those weights over the input. Note that if this code is rerun after the evaluation phase, the last minibatch will change. As evaluation and training share the same workspace.
```

Comment:
For convolutional models we can also see how they "think", i.e. which features they come up with. Instead of fetching learned weights, which can make less sense to a human, we fetch results of convolving those weights over the input. Note that if this code is rerun after the evaluation phase, the last mini-batch will change, since evaluation and training share the same workspace.